README.md

```diff
@@ -43,6 +43,37 @@ The services provided are:
 * `api` - convenience environment scoped to the `api` gem in the `/app/api` directory.
 * `sdk` - convenience environment scoped to the `sdk` gem in the `/app/sdk` directory.
 
+## Quick Start
+
+```ruby
+require 'opentelemetry/sdk'
+
+# Set preferred tracer implementation:
+SDK = OpenTelemetry::SDK
+
+factory = OpenTelemetry.tracer_factory = SDK::Trace::TracerFactory.new
+factory.add_span_processor(
+  SDK::Trace::Export::SimpleSpanProcessor.new(
+    # TODO: opentelemetry-python uses ConsoleSpanExporter:
```

Comment:
I would be ok moving the ConsoleSpanExporter from #131 into the SDK. What do you think @fbogsany?

Reply:
WFM 👍

Reply:
@mwear I've added a ConsoleSpanExporter, for review.